### ISSUE TITLE
Code formatting ("Format Document") got broken in  1.9.0

### ISSUE THREAD

**Issue by roypardi** [Microsoft/vscode#20169]
I updated to v 1.9.0 a few days ago and since then code formatting has been broken in html files which include javascript.

- VSCode Version:  1.9.0 (1.9.0)
- OS Version: OSX 10.11.6

Steps to Reproduce:

1. Context-menu "Format Document" or Shift-Option-F

<img width="784" alt="screen shot 2017-02-07 at 10 51 35 pm" src="https://cloud.githubusercontent.com/assets/9053196/22723076/759af0ae-ed89-11e6-9c04-a49321a571e9.png">

2. This is the result. If I use Command-Z I can toggle back and forth. I have only two extensions enabled: "1337 Theme" and "Material Icon Theme". I disabled both of them, reloaded the window and get the same result. Same for quitting and restarting VSC. I included the "suspect" html file below. Formatting worked fine in the prior VSC version.

<img width="1008" alt="screen shot 2017-02-07 at 10 51 55 pm" src="https://cloud.githubusercontent.com/assets/9053196/22723080/7d0ee7be-ed89-11e6-81c9-9ed1ba020a16.png">

```<!DOCTYPE html>
<html>

<header>
  <title></title>
  <link href="../node_modules/video.js/dist/video-js.min.css" rel="stylesheet">
  <!-- Insert this line above script imports  -->
  <script> if (typeof module === 'object') { window.module = module; module = undefined; }</script>

  <script src="../node_modules/video.js/dist/video.min.js"></script>
  <script src="../node_modules/videojs-wavesurfer/videojs.wavesurfer.js"></script>

  <!-- Insert this line after script imports  -->
  <script> if (window.module) module = window.modle;</script>
  


  <style>
    html,
    body {
      height: 100%;
      overflow: hidden;
      background-color:#2C3E50;
      margin: 0;
    }
    
    #mainContainer {
      position: absolute;
      top: 0;
      bottom: 0;
      left: 0;
      right: 0;
      overflow: hidden;
    }
  </style>


</header>


<body onload="onLoadApp()">
  
       <div class="centerVertically">
        <video id="videoPlayer" class="video-js vjs-default-skin vjs-16-9" controls preload="auto">
        </video>

  </div>

</body>

<script>
        // this html file is loaded as an Iframe.
        // when it is done loading it asks the parent for 
        // asset paths
        function getParamValue(paramName) {
            var url = window.location.search.substring(1);
            var qArray = url.split('&');
            for (var i = 0; i < qArray.length; i++) {
                var pArr = qArray[i].split('=');
                if (pArr[0] == paramName)
                    return pArr[1];
            }
        }

        // grap the video & poster frame refs from url
        var videoSrc = getParamValue('videoSrc');
        var videoPath = this.parent.getVideoPath(videoSrc);
     
        var poster = getParamValue('poster');
        var posterPath = this.parent.getImagePath(poster);

        videojs("videoPlayer", {}, function () {
            this.src(videoPath);
            this.poster(posterPath);
            this.load();
        });
    </script>
</html>

**Comment by mjbvz** [comment]
I believe this is a duplicate of  #19813

@aeschli Has a fix checked in to master and we plan on releasing a 1.9 recovery build with the fix as well

**Comment by aeschli** [comment]
@mjbvz Yes, the fix is in both master and 1.9, so the fix can be verified in the [insiders build](https://code.visualstudio.com/insiders).
